DigitalLibraryofTennessee/dltn_ingestion3_config_generator
# New sets added, two updated, and one remove (humbug) for Dec DPLA ing…

## What does this PR do?

It adds new sets for the December DPLA ingest, updates two set names, and removes one set. We decided to remove humbug from our Digital Collections, since these issues are also present in the Daily Beacon Collection. Volunteer Voices and TDH needed to be updated following a removal request from Washington County Archives for some items. Most additions are coming from UTC.

## How should this be tested?

Check and see if the set names match the set names in this issue - https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/316 (and that they also match the set names in Repox)
